.github/workflows/publish-theme.yaml

```
@@ -45,32 +46,27 @@ jobs:
                     cache: 'npm'
                     cache-dependency-path: 'package-lock.json'
                     always-auth: 'true'
-                    registry-url: 'https://npm.pkg.github.com/'
-                    scope: '@apify-packages'
-                env:
-                    NODE_AUTH_TOKEN: ${{ secrets.APIFY_SERVICE_ACCOUNT_NPM_TOKEN }}
 
             -   name: Setup git user and npm
                 run: |
                     git config --global user.name "Apify Release Bot"
                     git config --global user.email "noreply@apify.com"
 
-                    echo "access=public" > .npmrc
-                    echo "//registry.npmjs.org/:_authToken=${{ secrets.APIFY_SERVICE_ACCOUNT_NPM_TOKEN }}" >> .npmrc
+                    echo "access=public" > ~/.npmrc
+                    echo "@apify-packages:registry=https://npm.pkg.github.com/" >> ~/.npmrc
+                    echo "//registry.npmjs.org/:_authToken=${{ secrets.APIFY_SERVICE_ACCOUNT_NPM_TOKEN }}" >> ~/.npmrc
+                    echo "//npm.pkg.github.com/:_authToken=${{ secrets.NPM_TOKEN }}" >> ~/.npmrc
```

Please use this one instead, to make it more explicit which token it is. We'll eventually get rid of all the badly named tokens across the organization.

```suggestion
                    echo "//npm.pkg.github.com/:_authToken=${{ secrets.APIFY_SERVICE_ACCOUNT_GITHUB_TOKEN }}" >> ~/.npmrc
```

---

Note that the `NPM_TOKEN` is a repo override, its not even the one from apify org, I think it might be mine, I recall things got broken with the token that was in the org some time ago. We should clear this up finally and remove the repo override from here.

---

Yeah, the "if it works, don't touch it" rule went a long way here... this token is now used in all the workflows in this repo that access NPM in some way 😬  I'm glad it's changing though.